download/index.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: maindownloadpage
 title: Download
-release_version: 2.11.5
+release_version: 2.11.6
 release_date: "January 14, 2015"
 other_releases: [
-  ["maintenance_version", "Current 2.9.x maintenance release", 2.9.3, "February 28, 2013"],
-  ["maintenance_version", "Current 2.10.x maintenance release", 2.10.4, "March 24, 2014"],
-  ["development_version", "Current 2.11.x release", 2.11.5, "January 14, 2015"]
+  ["maintenance_version", "Last 2.9.x maintenance release", 2.9.3, "February 28, 2013"],
+  ["maintenance_version", "Last 2.10.x maintenance release", 2.10.5, "March 5, 2015"],
```

Comment:
Very true. Let's leave it. ("Ultimate" release doesn't have the right connotation!)